docs/user/dashboard/lens-advanced.asciidoc

```diff
@@ -295,6 +295,38 @@ image::images/lens_advanced_5_2.png[Line chart with cumulative sum of orders mad
 
 . Click *Save and return*.
 
+[discrete]
+[[compare-time-ranges]]
+=== Compare time ranges
+
+*Lens* allows you to shift individual dimensions in time to compare the currently selected time range with historical data. 
+To compare current sales numbers with sales from a week ago, follow these steps: 
+
+. Open *Lens*.
+
+. From the *Chart Type* dropdown, select *Line*.
+
+. From the *Available fields* list, drag and drop *Records* to the visualization builder.
+
+. Copy the *Count of Records* series by dragging it to the empty drop target of the *Vertical axis* dimension group (*Drop a field or click to add*)
+
+. Shift the second *Count of Records* series by one week to do a week-over-week comparison
+
+.. Click the new *Count of Records [1]* dimension
+
+.. Click *Add advanced options* below the field selector
+
+.. Click *Time shift*
+
+[role="screenshot"]
+image::images/lens_time_shift.png[Line chart with week-over-week sales comparison]
```

Comment:
Can you crop the screenshot to only focus on the configuration panel? The rest of the UI changes more often.